src/input_common/settings.h

```diff
@@ -66,6 +66,21 @@ constexpr int NUM_STICKS_HID = NumAnalogs;
 extern const std::array<const char*, NumAnalogs> mapping;
 } // namespace NativeAnalog
 
+namespace NativeMotion {
+enum Values {
```

Comment:
```suggestion
enum Values : int {
```

otherwise the forward declaration in main.h won't work.